### PR TITLE
add sdp as input for tangentBC

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description

tangentBC: add anoption to read sdp points as input.

# Checklist

- [ ] Doxygen documentation of the code completed (classes, methods, types, members...).
- [ ] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [ ] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [ ] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [ ] Update the readme with potentially a screenshot of the tools if it applies. 
- [ ] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -26,7 +26,8 @@
 - *estimators*:
     - 2dlocalEstimators: add an option to export the generated contour.
      (Bertrand Kerautret, [#285](https://github.com/DGtal-team/DGtalTools/pull/285))
-    
+    - tangentBC: add anoption to read sdp points as input.
```
Comment (dcoeurjo) [inline ChangeLog.md]: an option

Comment (kerautret) [inline ChangeLog.md]: done thanks

**Comment by kerautret** [comment]
and I have also added the source contour point as output (can be important when using fc)

**Comment by dcoeurjo** [comment]
Ok thanks. Let's wait for travis.

Are you ready for the release BTW ?

**Comment by kerautret** [comment]
yes I have not done the DGtal doc example move (perhaps time tonight)

**Comment by kerautret** [comment]
oups I just forget a white space in the output, I pushed it sorry for travis

**Comment by kerautret** [comment]
fine done merging thanks

**Pull request merged**